src/manifest.ts

```diff
@@ -0,0 +1,62 @@
+import { CrxPlugin } from '@crxjs/vite-plugin';
+import { version } from '../package.json';
+
+type ManifestV3 = Parameters<NonNullable<CrxPlugin['transformCrxManifest']>>[number];
+
+const manifest: ManifestV3 = {
+  manifest_version: 3,
+  name: 'Wildhero - Plants Trees With Your Email',
+  short_name: 'Wildhero',
+  description: 'Plants trees with your email.',
+  version,
+  key: 'MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0NulF4+ocWkFI/mDDyjG5v1+Bj7Sa6qceNkRlOGjNgPIWqz7EJeYxCl96Twf8nMUMGXAXTJ5AFoq5Hcf+rsQb/h/84o81XMCK8DLr5D3cGyAN0bE3eAP8rlUUWWwS3EiB2yxeEkfVGEFP1YMVxDXE1k0+NOe361iAD53CpakGGoeKhn+Nr0SRonK2239r5pGF+7yKML1UywbvgdoLu02TNI1OmGTHDZ+9XCYFG4gaWGUe/3XFi2AMgJD3RI5RvVf2z5j3TvZafnhT181otxEcVQPs2Et8U6yDedYV/nP0gKfazP4EoJio8MTcM6bzdsRRrHdd6qcAPe66DAj2VLMAQIDAQAB',
```

Comment:
This can be a security risk if exposed or misused. Typically, keys are managed securely and not included in public files.
**Suggestion:** For production, manage keys securely outside of the codebase, or use Chrome’s Developer Dashboard for key management.